src/posts/2021-09-03-getting-started-with-orca.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Getting started with Orca screen reader on Gnome desktop environment on Ubuntu 20.04 LTS
+description: A quick overview of getting started with Orca on Gnome desktop environment on Ubuntu 20.04 LTS.
+category: Assistive technology
+author: Joseph mawa
+date: 2021-09-03
+further_reading:
+  - title: Orca screen reader documentation
+    url: https://help.gnome.org/users/orca/stable/index.html.en
+    source: Gnome foundation
+  - title: Orca screen reader tutorial
+    url: https://techblog.wikimedia.org/2020/07/02/an-orca-screen-reader-tutorial/
+    source: Wikimedia foundation
+tags:
+  - assistivetechnology
+---
+
+## Introduction to Orca
+
+Orca is a free, open-source screen reader developed under the GNOME project. Several Linux distributions have it installed by default.
+
+This article will introduce you to the basics of Orca on Gnome desktop environment on Ubuntu version 20.04 LTS. There might be some slight variations if you are using a different desktop environment or using GNOME on a different Linux distribution. You need to consult the manual for your Linux distribution or desktop environment.
```

Comment:
Similarly, can we link to a good explainer of what Ubuntu is?

Reply:
I have made `Ubuntu` hyperlink too.